main/src/ca/uwaterloo/flix/language/phase/Typer.scala

```diff
@@ -1595,6 +1595,8 @@ object Typer {
           resultEff = eff
         } yield (constrs, resultTyp, resultPur, resultEff)
 
+      case KindedAst.Expression.Par(exp, tvar, pvar, evar, loc) => ???
```

Comment:
Here we should just call recursively on exp and return its type, purity, effect.